google-cloud-container/lib/google/cloud/container/v1/cluster_manager_client.rb

```diff
@@ -47,7 +47,7 @@ class ClusterManagerClient
           DEFAULT_SERVICE_PORT = 443
 
           # The default set of gRPC interceptors.
-          GRPC_INTERCEPTORS = []
+          GRPC_INTERCEPTORS = [].freeze
```

Comment:
We don't want to freeze this. This is where users configure interceptors to use.